integration-tests/src/test/java/org/jboss/pnc/bacon/test/pig/PigFunctionalTest.java

```diff
@@ -29,13 +30,15 @@ static String init(Path configDir, boolean clean, Optional<String> releaseStorag
         String suffix = prepareSuffix();
         // todo release storage url mocking
         if (configDir == null) {
-            throw new FatalException("You need to specify the configuration directory!");
+            throw new RuntimeException("You need to specify the configuration directory!");
         }
+        System.setProperty(PIG_CONTEXT_DIR, targetDirectory.toString());
```

Comment:
I think tests that set system properties should use https://github.com/stefanbirkner/system-lambda (https://github.com/stefanbirkner/system-rules for JUnit4)